### ISSUE TITLE
Structural error at "security". it should be object

### ISSUE THREAD

**Issue by lohmuller** [DarkaOnLine/L5-Swagger#463]
- L5-Swagger Version: 8.3.0
- PHP Version: 8.0.15
- OS: Debian 10

### Description:

By default in swagger, in the security column should be an array of objects or an empty array.
After starting to use **operationId** (to use as path) + **openApi Security** in my project, the **security** field in json file is generating an empty array inside and generating some warnings/errors in some swagger viewers

The wrong security exported in the json file:
```
...
{
    "security": [
        []
    ]
}
...
```

it should be an ``` "security": [ {} ] ``` (array of object) or ``` "security": [ ] ```  (empty array)

### Steps To Reproduce:

Adding ```@OA\SecurityScheme``` 

and in a controller (or anywhere else you want):
 ```security={{ "thatSecurityName": {} }}```
  + 
  ``` operationId="Modules\Namespace\ControllerName::ActionExample", ```

**Comment by lohmuller** [comment]
One of my ugly fix was to change the I5-swagger.php, adding the ```(object)``` before the security array example

```php 
 'security' => [
                /*
                 * Examples of Securities
                */
                (object) [
                    /*
                    'oauth2_security_example' => [
                        'read',
                        'write'
                    ],

                    'passport' => []
                    */
                ],
            ],
            
```

So even if it generates an empty security, it will be an empty object in the json, not more an array

**Comment by DarkaOnLine** [comment]
This should fix the issue: #460 -  https://github.com/DarkaOnLine/L5-Swagger/pull/460/commits/86c5f9db1de82f2a0b716d1d9cface862a798d56
